testsuite/features/secondary/min_bootstrap_negative.feature

```diff
@@ -67,6 +68,21 @@ Feature: Negative tests for bootstrap normal minions
      And I select the hostname of "proxy" from "proxies"
      And I click on "Bootstrap"
      And I wait until I see "Successfully bootstrapped host!" text
+     And I navigate to "rhn/systems/Overview.do" page
```

Comment:
Personally, I'd prefer to use a UI step instead of providing a url endpoint here. Just because for me the tests must be near as possible as what a customer will do, meaning clicking options in the UI. But it might just be a matter of taste.

Reply:
Right. It's not a matter of taste, but a generic rule: **do not depend on internals**.

I think we should even remove the `I navigate to` step definition.

But this is probably for another PR. This should be addressed globally. Maybe create a card so it does not get lost.

Reply:
Oki. Anyway, my suggestion here is to just use that step instead:
```
When I follow the left menu "Home > Overview"
```